service/mapper/transitions.go

```diff
@@ -247,26 +426,23 @@ func (t *Transitions) CollectRegisters(s *State) error {
 
 	// At this point, we have collected all the payloads, so we go to the next
 	// step, where we will index them.
-	s.status = StatusCollected
-
+	s.status = StatusMap
 	return nil
```

Comment:
I'm beginning to think that this was not an involuntary change and that you decided to specifically cuddle the returns with setting the status for the next transition. Is that right? I think I can see your reasoning, but by the code style standard that I usually follow, a return statement should never be cuddled in such a case. It looks out of place 😬